backend/src/test/TestHelper.ts

```diff
@@ -5,6 +5,7 @@ import { VoterAuth } from "../../../domain_model/VoterAuth";
 import makeApp from "../app";
 import Logger from "../Services/Logging/Logger";
 import { TestLoggerImpl } from "../Services/Logging/TestLoggerImpl";
+import ServiceLocator  from "../__mocks__/ServiceLocator"
```

Comment:
pull in the actual ServiceLocator instead of the mock to fix the problem with the test
import ServiceLocator  from "../ServiceLocator"

JEST is using some magic to swap out the ServiceLocator with the Mock.   As far as I can tell it creates a "mock-instance" of the Service Locator, which is different than when you pull in the mock SL directly...

I wouldn't mind to go in and make this explicit instead of relying on JEST voodoo, but not tonight :P

Reply:
This fixed it, thanks! I forgot about the Jest mock voodoo. The jest mocks don't behave quite like I'd expect so I wouldn't mind swapping them out somehow. I was thinking this would be a good case for dependency injection but you probably know some javascript voodoo to do it.